### ISSUE TITLE
luarocks.build.command does not work at all on windows

### ISSUE THREAD

**Issue by Frityet** [luarocks/luarocks#1535]
### Please read the following before submitting:
- Please do NOT submit bug reports for questions. Ask questions in the [Gitter
  chat-room](https://gitter.im/luarocks/luarocks).
- Please do NOT submit duplicate bug reports. Look through the [issue
  tracker](https://github.com/luarocks/luarocks/issues) before submitting.

### Please fill out the following:
- **Platform:**
  - Windows

- **LuaRocks version:**
  - 3.9.2
- **Configuration file:**
```lua
rocks_trees = {
    { name = [[user]],
         root    = home..[[/luarocks]],
    },
    { name = [[system]],
         root    = [[C:\Program Files\Lua\Modules]],
    },
}
variables = {
    MSVCRT = 'm',   -- make MinGW use MSVCRT.DLL as runtime
    LUALIB = 'lua51.dll',
    CC = [["C:\Program Files\LLVM\bin\gcc.exe"]],
    MAKE = [["C:\Program Files\LLVM\bin\make.exe"]],
    RC = [["C:\Program Files\LLVM\bin\windres.exe"]],
    LD = [["C:\Program Files\LLVM\bin\gcc.exe"]],
    AR = [["C:\Program Files\LLVM\bin\ar.exe"]],
    RANLIB = [["C:\Program Files\LLVM\bin\ranlib.exe"]],
    --MD5SUM = [["C:\Program Files (x86)\LuaRocks\tools\md5sum.exe"]]
    MD5SUM = [[md5sum]],
    WGET = [[wget]],
}
verbose = false   -- set to 'true' to enable verbose output

```
- **LuaRocks output from when the issue occurred:**
  - Use the `--verbose` flag.\
 https://gist.github.com/Frityet/0f9b12c792aaf453a0f7648d08f5b1e4

- **Description:**
  - The steps you took in plain English to reproduce the problem.

Make rockspec with `command` type build and try and use it on windows

**Comment by hishamhm** [comment]
This should have been fixed by https://github.com/luarocks/luarocks/pull/1569

**Comment by Frityet** [comment]
Amazing! thank you so much
